### PR TITLE
Gatsby remark graphviz

### PR DESCRIPTION
**Pull request by Moocar**
Found a bug where the top level gatsby-remark-graphviz promise was never being resolved, causing createPages to freeze if more than one remark plugin was configured.

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit f9c79ec76718e423b2c40877ab8170683eb3fa0c

https://app.netlify.com/sites/using-postcss-sass/deploys/5b750c8ac6aed61ff53e1f2a

**Comment by KyleAMathews** [comment]
Deploy preview for *using-glamor* failed.

Built with commit f9c79ec76718e423b2c40877ab8170683eb3fa0c

https://app.netlify.com/sites/using-glamor/deploys/5b750c8bc6aed61ff53e1f33

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit f9c79ec76718e423b2c40877ab8170683eb3fa0c

https://app.netlify.com/sites/using-postcss-sass/deploys/5b750d19c965920bffd6bced

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit 6754bbe00f3a1d0a139832560a33cd43d739f496

https://app.netlify.com/sites/gatsbyjs/deploys/5b751059c6aed629e13e1ef7

File: packages/gatsby-remark-graphviz/README.md [hunk 1]
```diff
@@ -38,6 +38,10 @@ Which will be rendered using viz.js and the output html will replace the code bl
 
 ![rendered-graph](./rendered-graph.svg)
 
+## Caveats
+
+In your gatsby-config.js, make sure you place this plugin before other remark plugins that modify code blocks (like prism).
```
Comment (m-allanson) [inline packages/gatsby-remark-graphviz/README.md]: This is awkward but I can't think of any way to avoid it 🤔

**Pull request merged**